packages/fxa-settings/tailwind.config.js

```diff
@@ -12,6 +12,17 @@ module.exports = {
     whitelist: [],
   },
   theme: {
+    extend: {
+      padding: {
+        7: '1.75rem',
+      },
+      margin: {
+        11: '2.75rem',
+      },
+      borderRadius: {
+        xl: '.75rem',
+      },
+    },
```

Comment:
Just checking -- all of these are in addition to and not overriding existing classes, right?

Reply:
Correct, anything under `extend` extends the theme 👍